src/libs/actions/Session.js

```diff
@@ -73,44 +137,60 @@ function signIn(partnerUserID, partnerUserSecret, twoFactorAuthCode = '', exitTo
 
                     setSuccessfulSignInData(createLoginResponse, exitTo);
 
-                    if (credentials && credentials.login) {
-                        // If we have an old login for some reason, we should delete it before storing the new details
+                    // If we have an old login for some reason, we should delete it before storing the new details
+                    if (credentials.login) {
                         API.DeleteLogin({
                             partnerUserID: credentials.login,
                             partnerName: CONFIG.EXPENSIFY.PARTNER_NAME,
                             partnerPassword: CONFIG.EXPENSIFY.PARTNER_PASSWORD,
                             doNotRetry: true,
                         })
-                            .catch(error => Onyx.merge(ONYXKEYS.SESSION, {error: error.message}));
+                            .catch(console.debug);
                     }
 
-                    Onyx.merge(ONYXKEYS.CREDENTIALS, {login, password});
+                    Onyx.merge(ONYXKEYS.CREDENTIALS, {password});
```

Comment:
I _think_ removing `login` from here means that we are no longer saving the generated login we created and are also just saving the original password because of the variable name was changed. But not sure if there was some other plan at work